content/en/cloud_cost_management/budgets.md

```diff
@@ -10,28 +10,62 @@ further_reading:
 ## Overview
 After you start ingesting your costs in Cloud Cost Management, set up budgets and visualize how you are tracking against budgets.
 
-## Create a budget
+You can create two types of budgets:
+
+- [**Basic**](#create-a-basic-budget): A flat, single-level budget for tracking your cloud costs.
+- [**Hierarchical**](#create-a-hierarchical-budget): A two-level, parent-child budget for tracking costs in a way that mirrors your organization's structure. For example, you can budget on a team within a department tag so that you can understand your budget health performance at the department (parent) and team (child) levels. In addition, this option allows you to create a single budget instead of needing to create multiple budgets.
```

Comment:
```suggestion
- [**Hierarchical**](#create-a-hierarchical-budget): A two-level, parent-child budget for tracking costs in a way that mirrors your organization's structure. For example, you can budget for a team within a department tag to track budget health at both the department (parent) and team (child) levels. In addition, this option allows you to create a single budget instead of needing to create multiple budgets.
```